wire-ios/Wire-iOS/Sources/ForegroundNotificationFilter.swift

```diff
@@ -62,6 +62,7 @@ extension ForegroundNotificationFilter: ForegroundNotificationResponder {
 
         // conversation view is visible for another conversation
         let svc = clientVC.mainSplitViewController // TODO: [WPB-11994] test this flow manually
+
```

Comment:
these empty lines resulted from the conflict resolution, shall we remove them?

Reply:
Will remove!